qcodes/instrument/channel.py

```diff
@@ -263,6 +263,18 @@ def append(self, obj: InstrumentChannel):
         self._channel_mapping[obj.short_name] = obj
         return self._channels.append(obj)
 
+    def remove(self, obj: InstrumentChannel):
```

Comment:
no testing?

Reply:
It should probably have a test but was not ready to merge anyway. Somehow got pushed to master anyway by accident with failing test

Reply:
I added extensive testing and this should be good to go now, @jenshnielsen, @astafan8

Reply:
👍